src/Uno.UI/UI/Xaml/NativeApplication.cs

```diff
@@ -55,19 +55,29 @@ public override void OnCreate()
 
 		private void OnActivityStarted(Activity activity)
 		{
-			_app.InitializationCompleted();
-			if (_lastHandledIntent != activity.Intent &&
-			    activity.Intent?.Extras?.ContainsKey(JumpListItem.ArgumentsExtraKey) == true)
-			{
-				_lastHandledIntent = activity.Intent;
-				_app.OnLaunched(new LaunchActivatedEventArgs(ActivationKind.Launch, activity.Intent.GetStringExtra(JumpListItem.ArgumentsExtraKey)));
-			}
-			else
-			{
-				_app.OnLaunched(new LaunchActivatedEventArgs());
-			}
+			_app.InitializationCompleted();
+
+			string packagePrefix = Android.App.Application.Context.PackageName; // tego moze nie byc?

```

Comment:
Can you use English for the comments please ?

Reply:
Of course. Sorry for leaving this in Polish.